From 6e15d672dc31a83fb1ac64c7dae2e24b7d3526df Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Fri, 19 Jun 2015 19:20:39 +0200
Subject: [PATCH 50/64] google_now: disable this

"Google Now is an intelligent personal assistant developed by Google.
It is available within the Google Search mobile application for
Android, as well as the Google Chrome web browser on personal
computers." (-WP)

Get off my lawn!
---
 chrome/common/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 653e7692e33d..643c412da672 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -35,7 +35,7 @@ buildflag_header("features") {
     "ENABLE_BACKGROUND=$enable_background",
     "ENABLE_BASIC_PRINT_DIALOG=$enable_basic_print_dialog",
     "ENABLE_CAPTIVE_PORTAL_DETECTION=$enable_captive_portal_detection",
-    "ENABLE_GOOGLE_NOW=$enable_google_now",
+    "ENABLE_GOOGLE_NOW=0",
     "ENABLE_HANGOUT_SERVICES_EXTENSION=$enable_hangout_services_extension",
     "ENABLE_ONE_CLICK_SIGNIN=$enable_one_click_signin",
     "ENABLE_NATIVE_NOTIFICATIONS=$enable_native_notifications",
-- 
2.15.0

